index.d.ts

```diff
@@ -11,7 +11,8 @@ export interface apiObject {
     | boolean
     | undefined
     | apiObject
-    | (string | number | boolean | apiObject)[];
+    | Date
+    | (string | number | boolean | apiObject | Date)[];
```

Comment:
Let's also support `null` and `undefined` values.

```suggestion
    | null
    | Date
    | (string | number | boolean | apiObject | Date | null | undefined)[];
```

Reply:
Addressed